src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
Since you are using the React 18 version there is no need to import React, so you can remove it